tests/test_delb_plugin.py

```diff
@@ -1,6 +1,6 @@
 import pytest  # type: ignore
 
-from delb import Document, FailedDocumentLoading
```

Comment:
this should have worked though, but i have no clue.

Reply:
this seems also to be the error yielded in `test_exist_client` and i can't reproduce the behaviour locally.

Reply:
ah, no, i can reproduce if i invoke the tests correctly.

Reply:
i added an ignore annotation here.